_data/simple-icons.json

```diff
@@ -5423,11 +5428,6 @@
             "hex": "FF1E0D",
             "source": "http://www.makerbot.com/makerbot-press-assets"
         },
-        {
-            "title": "MAMP",
-            "hex": "02749C",
-            "source": "https://www.mamp.info/en/mamp/mac/"
-        },
```

Comment:
This removal might be a mistake?

Reply:
I suppose. I don't remember this icon. 
Restored it from develop.